### DISCUSSION TITLE
Non-academic reference grid / standard circle r100

### DISCUSSION THREAD

**Discussion by VolksNav** [OvertureMaps/data#318] [Discussion]
Average people think they aren't able to read maps [www.volksnav.de/Mapdanger](http://www.volksnav.de/Mapdanger) . 
The truth is, maps have too little interface to the reality. The question "where?" is answered with "here!" [www.volksnav.de/YouAreHere](http://www.volksnav.de/YouAreHere)
Even a simple standard circle can change this unwanted situation [www.volksnav.de/aSimpleCircle](http://www.volksnav.de/aSimpleCircle).

It is proposed to think about integrating reference grids on Overture.

**Comment by mprioleau** [comment]
Thanks for reaching out.  Overture's primary metrics are around adoption by map service providers.  The roadmap will be determined by the interests and needs of the Overture community and the market. 
Has this reference system been adopted by the broader mapping market? That would directly drive our interest in using it in Overture.

**Comment by VolksNav** [comment]
Thank you, Marc.
The grid is only the peak of iceberg, that means, is an Overture to completely new possibilities for cartographers:

1 - developers of smart indoor, urban and global location codes, much better than the alternatives [www.volksnav.de/alternatives](http://www.volksnav.de/alternatives).  

For example, ESRI/theScienceOfWhere promotes location codes based on names (??) [www.volksnav.de/w3wComparison](http://www.volksnav.de/w3wComparison) . 
Will Overture simply ignore this demand or promote other enigmatic codes?

2 - developers of imaginable digital addresses and modern house numbers which consider the street configurations

3 - developers of a better signage and pictograms which also indicate the directions north, downtown, ALL emergency exits etc. etc.

4 - developers of apps [www.volksnav.de/Smartphone](http://www.volksnav.de/Smartphone)  

and so on. The invention was developed to form an orientation standard [https://act4sdgs.org/profile/SDG18](https://act4sdgs.org/profile/SDG18) , so I'm proposing to UNIDO Vienna to form the standard. For example UN Habitat would offer the system directly to communities. The start would be in Cameroon, where Minpostel just asked for digital addresses and got much more [www.volksnav.de/UNIDOyaounde](http://www.volksnav.de/UNIDOyaounde)  

The Overture way would be much faster, so I propose a similar partnership/monopole to your community. Besides: I live in the Oktoberfest city.

**Comment by VolksNav** [comment]
Another important detail: 
Overture would have the comfortable possibility to publish that the goal is not to launch “just another map”, but mainly to make people get smarter! 
The division of the horizon into 12 cardinal points is used by soldiers, boy scouts, the blind, pilots etc. for more than 120 years. Among many benefits, this method minimizes right/left and east/west confusions   [www.volksnav.de/confusions](http://www.volksnav.de/confusions)

Overture would demonstrate courage and pioneer spirit implementing this important orientation tool although nobody asked for it!  Why? Because it is only possible to ask for something knowing that it exists and that it can be implemented on maps.
  
Besides: I’m also proposing the system to E. Musk etc.

**Comment by RobSoetewey-TomTom** [comment]
Hi, 
Can you maybe elaborate on how your system is supposed to be embedded in a map? Scanning through the referred pages, my understanding is that it's main use is relative to a users current position and heading rather than having absolute and fixed elements in a map. In the first case, this would be the domain of map implementations rather than something that can be embedded in the map. Am I missing something?

Best regards,
Rob

**Comment by VolksNav** [comment]
Yes, Rob, you're possibly thinking too complicated.

Like you'd overlay your yesterday's route on a map, the proposed reference grid is simply an optional overlay of STANDARD lines, see for example your location in Gent [www.volksnav.de/Gent](http://www.volksnav.de/Gent). The lines should facilitate the reading of the map for those who still can read the clock, still know something about cardinal points, distances etc. The State of the Map is to show HERE!, sometimes accompained by enigmatic codes and enigmatic grids. 

Under this iceberg are swimming the death of the obsolete postcodes, enigmatic building or room numbers etc. and I'd appreciate if Overture Maps would also think outside of the map. The bigger, the easier to understand.

**Comment by VolksNav** [comment]
Underlining the goal to make people become smarter, it is also proposed to exterminate the VMB  Vertical Maps Bias, see 
 https://www.esri.com/arcgis-blog/products/product/mapping/map-orientation-when-true-north-is-not-at-the-top. North is not above / south is not below, a simple STANDARD pictogram neutralizes this  discrepancy.

It is also proposed to jump from theory to for example an actual nationwide demand in Cameroon, if possible with inclusion of the UN [www.volksnav.de/UNIDOyaounde](http://www.volksnav.de/UNIDOyaounde). Feel free to ask Minpostel for the final report.

**Comment by VolksNav** [comment]
The following new situation should help to understand the value of the invention:

I’ve learned to know Dr. Massoud Harun-Mahdavi, a former Mayor of Meshhad, Iran. 
He lives here in Munich and – other than Overture - understood immediately the superiority of imaginary clocks, respectively 12 cardinal points. He’s taking measures for an immediate start with station and stop point codes [www.volksnav.de/Meshhad](http://www.volksnav.de/Meshhad).
Such application would belong to your themes Places and also Transportation. 
Wouldn’t it be wonderful to have a language-independent and imaginable answer to the question where? 
Remark: ESRI/theScience ofWhere answers with 3 words (?) in 50 languages.

**Comment by VolksNav** [comment]
Please be informed that the Cameroon's capital city has made an Overture: the following innovations have been defined for the bus route Olembé-Ahala:

- line codes 6>12 and 12>6 instead of one aleatory number
- imaginable stop point codes acc. to www.volksnav.de/Yaounde 
- line pictograms as small maps which include the proposed r100 reference circle to Overture Maps  

These overture data are mentioned on my GTFS solicitation https://github.com/google/transit/issues/404 and have the chance to be mentioned on history books as world's first imaginable line and location codes. All transport organizations have been informed.

I'm confident that this new situation will be a motivation for Overture Maps to continue the  discussions.

**Comment by VolksNav** [comment]
Address for all (AFA) is a new contributor member of Overture Maps. This confirms that the demand for better answers to the simple question “where?” really exists.

Like other location codes [www.volksnav.de/alternatives](http://www.volksnav.de/alternatives), AFA has the following characteristics: 

- is just another enigmatic alternative to lat/lon, having no chances to become a standard. 

- has no benefic influence on cartography, signage or navigation.

- doesn’t consider the street configurations, consequently is not an addressing system but proper for flying searches

- is not proper for logical search engines [www.volksnav.de/search](http://www.volksnav.de/search) 

I renew herewith my proposal to Overture Maps, thank you in advance for a discussion in favor of the consumer.

Besides: Dubai [www.volksnav.de/Dubai](http://www.volksnav.de/Dubai) and/or Osaka [www.volksnav.de/Osaka](http://www.volksnav.de/Osaka) will very probably be world’s Overture to the Munich Orientation Convention.
